.template/lib/thor_utils.rb

```diff
@@ -0,0 +1,12 @@
+class ThorUtils
+  def self.ignore_tt
+    # NOTE: change template extension so it would skip
+    #       `when /#{TEMPLATE_EXTNAME}$/` condition and
+    #        fallback to default `copy_file`
+    Thor::TEMPLATE_EXTNAME.concat '_no_match' # => .tt_no_match
+    yield
+  ensure
+    # NOTE: make sure to undo our dirty work after the block
+    Thor::TEMPLATE_EXTNAME.chomp! '_no_match' # => .tt
+  end
+end
```

Comment:
missing empty ending line 😇

Reply:
😅 I have added this new ending line in the latest changes already. Sorry about that 🤦